pints/toy/_lotka_volterra_model.py

```diff
@@ -38,13 +41,41 @@ class LotkaVolterraModel(pints.ForwardModel, ToyModel):
     References
     ----------
     .. [1] https://en.wikipedia.org/wiki/Lotka%E2%80%93Volterra_equations
```

Comment:
This could just say `https://en.wikipedia.org/wiki/Lotka-Volterra_equations`

Reply:
Without the %E2% etc

Reply:
changed